external/loaders/loaders/batches/_batch.py

```diff
@@ -115,6 +120,10 @@ def batches_from_mapper(
         needs_grid: Add grid information into batched datasets. [Warning] requires
             remote GCS access
         in_memory: if True, load data eagerly and keep it in memory
+        drop_nans: if True, drop NaN values from the data, and raise an
+            exception if all values in a batch are NaN
```

Comment:
Can you add to this to say it only operates if data is stacked?

Reply:
Done.